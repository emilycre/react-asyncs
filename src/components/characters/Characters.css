.Characters {
  display: flex;
  justify-content: space-around;
  flex-wrap: wrap;
  align-items: center;

  li {
    width: 20vw;
    margin: 2em;
    padding: 2rem;

    background-color: #f0f0f0;
    text-align: center;
    list-style-type: none;
  }

  section {
    margin: auto;
    animation: insert 0.5s linear;
  }

  img {
    width: 10rem;
    margin: auto;
  }

  @keyframes insert {
    from {
      width: 0;
    }

    to {
      width: 100%;
    }
  }
}
