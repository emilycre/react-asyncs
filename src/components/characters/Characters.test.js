import React from 'react';
import { shallow } from 'enzyme';
import Characters from './Characters';

describe('Characters Component', () => {
  it('Renders Characters', () => {
    const characters = [
      {
        name: 'Morty',
        status: 'Alive',
        species: 'Human',
        image: '/image'
      },
      {
        name: 'Summer',
        status: 'Alive',
        species: 'Human',
        image: '/image'
      }
    ];
    const wrapper = shallow(<Characters characters={characters} />);
    expect(wrapper).toMatchSnapshot();
  });
});
