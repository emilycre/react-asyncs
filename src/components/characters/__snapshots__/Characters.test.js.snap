// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Characters Component Renders Characters 1`] = `
<ul
  className="Characters"
/>
`;
